GDJS/Runtime/runtimescene.ts

```diff
@@ -507,7 +530,10 @@ namespace gdjs {
       if (renderDebugDraw && this._layersCameraCoordinates) {
```

Comment:
You should make this a boolean of the scene: `this._renderDebugDraw`.
That is set to true or false according to the the function `enableDebugDraw`. Same for the other booleans (or you can store them in the renderer, as you wish).